tensorflow/compiler/xla/tests/llvm_compiler_test.cc

```diff
@@ -28,6 +28,47 @@ limitations under the License.
 #include "tensorflow/stream_executor/stream_executor.h"
 
 namespace xla {
+namespace gpu {
+
+// Creating dummy data structure needed to initialize a GpuCompilerTest
+PLATFORM_DEFINE_ID(kDummyTestId);
+constexpr char kDummyTriple[] = "dummy-triple";
+constexpr char kDummyLayout[] = "e";
+
+// This class is is a dummy implementation of GpuCompiler and is targeted for
+// unit test only
+class GpuCompilerTest : public GpuCompiler {
```

Comment:
*Test is perhaps not a best name, since it is reserved for actual tests.

Reply:
I'm following the naming convention with rest of the test including: `CpuCompilerTest`, `GpuCompilerTest` and both of them are child class of `LLVMCompilerTest`. This is used directly in `TEST_F` for unit test as well.